### ISSUE TITLE
Accessible links in the documentation

### ISSUE THREAD

**Issue by leagrdv** [swisspost/design-system#5276]
Some of the links in the documentation are not really clear on where they lead to:

- [Introduction page](https://next.design-system.post.ch/?path=/docs/introduction--docs): "..., or click `here`." doesn't explain where "here" leads to.
- [Packages > Components](https://next.design-system.post.ch/?path=/docs/edfb619b-fda1-4570-bf25-20830303d483--docs): `CDN` link not clear that it goes to a wiki page
- [Packages > Internet header](https://next.design-system.post.ch/?path=/docs/cfa21cbc-5f8d-4a42-8165-7f6ed20bf133--docs): Full links are written, it'd be better to have a contextual text instead
- [Packages > Internet-header](https://next.design-system.post.ch/?path=/docs/cfa21cbc-5f8d-4a42-8165-7f6ed20bf133--docs): `tag` link
- [Foundations > Typography > Overview](https://next.design-system.post.ch/?path=/docs/677cfabf-dbf0-4de2-ad07-6d5bfb9e2375--docs): Links `link` and `invidunt ut labore et dolore` are dummy links, since they're not in the typical preview iframe, it might be nice to have it stated that they are dummy links.
- [Foundations > Layout > Grid](https://next.design-system.post.ch/?path=/docs/7240f2ef-216a-490e-9bd8-c0cef19f7b31--docs): "... examples of grid gutters `here`"
- [Components > Popover](https://next.design-system.post.ch/?path=/docs/9a636763-de2d-4f72-bc81-98daf10871f7--docs) and [Components > Tooltip](https://next.design-system.post.ch/?path=/docs/cd684d90-e7a7-41a9-8923-b1b72ad9b384--docs): Full link to the floating-ui, it should be contextual instead. Verify that this link is really necessary, as it simply leads to a page with a list that is already present in our docs
- [Components > Subnavigation](https://next.design-system.post.ch/?path=/docs/87ceabbb-f552-46eb-8a47-4d84e7f8cef0--docs): `palette` link not really clear
- [Components > Table](https://next.design-system.post.ch/?path=/docs/81799516-470c-446b-a049-54946b6ccfca--docs): `documentation` (should be clear that it links to primeng's table documentation)
- [Guideline > Metadata usage](https://next.design-system.post.ch/?path=/docs/a9543ea3-044d-4e86-bbb9-985c3f74f270--docs): All the links on this page are not really clear (e.g. `more information` (on?), `general information` (on?), `facebook` (which is a link to the og tags on facebook, and not just a link to facebook), `open graph`, `X`)
- [Migration guides](https://next.design-system.post.ch/?path=/docs/c23b1d0b-76b3-4e38-aa76-b10c29bb873f--docs): version links should specify what library it is (e.g. `5.1.x` link)
